src/metatrain/cli/train.py

```diff
@@ -118,22 +115,32 @@ def train_model(
     :param continue_from: File to continue training from.
     """
     ###########################
-    # LOAD ARCHITECTURE #######
+    # VALIDATE BASE OPTIONS ###
     ###########################
 
-    try:
-        architecture_name = options["architecture"]["name"]
-    except ConfigKeyError as exc:
-        raise ValueError("Architecture name is not defined!") from exc
+    # Training, test and validation set options are verified within the
+    # `expand_dataset_config()` function.
+
+    with open(PACKAGE_ROOT / "share/schema-base.json", "r") as f:
+        schema_base = json.load(f)
+
+    jsonschema.validate(instance=OmegaConf.to_container(options), schema=schema_base)
```

Comment:
We can wrap these two lines in a function `check_base_options`, but might be an overkill...